README.md

```diff
@@ -1,19 +1,53 @@
-Absolute
-=============
-
+# Absolute
 [![Linux/Max Build Status](https://travis-ci.org/lunchclass/absolute.svg?branch=master)](https://travis-ci.org/lunchclass/absolute/branches)
-[![Windows Build status](https://ci.appveyor.com/api/projects/status/099u4iekeny4lpsa/branch/master?svg=true)](https://ci.appveyor.com/project/romandev/absolute/branch/master)
+[![Windows Build status](https://ci.appveyor.com/api/projects/status/pmac6fwbclavu8n0/branch/master?svg=true)](https://ci.appveyor.com/project/romandev/absolute/branch/master)
 [![Coverage Status](https://coveralls.io/repos/github/romandev/absolute/badge.svg?branch=master)](https://coveralls.io/github/romandev/absolute?branch=master)
 
-### Welcome to Absolute!
+
+
+## Welcome to Absolute!
 
 Absolute is a web-based order platform like siren-order for Starbucks.
 We are focused on creating the platform that is portable, cheaper, easier,
 decentralized and appless.
 
-## Documentation
+**Portable:**  
+
+**cheaper:** 
+
+**easier:**  
+
+### Flow
+TODO Input Flow Image
+
+### Features
+* client(web): 
+* push server: 
+* payment: 
+
+## Contributing to Absolute
+Contributions to Absolute are welcomed and encouraged! Please see the
+[Contributing to Absolute guide](http://lunchclass.io).
```

Comment:
We need another file e.g. CONTRIBUTION.md

Reply:
I will delete it and make it contribution.md later.